kind/cmd/kind/create/create.go

```diff
@@ -45,6 +46,7 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.Flags().StringVar(&flags.Name, "name", "1", "the cluster name")
 	cmd.Flags().StringVar(&flags.Config, "config", "", "path to create config file")
+	cmd.Flags().StringVar(&flags.ImageName, "image", "", "name of the image to use for booting the cluster. If this is non-empty, it will override the value specified in the --config file.")
```

Comment:
might make this a little shorter later, but /shrug for now, it's certainly clear :-)